resolwe_bio/descriptors/unified-descriptor.yml

```diff
@@ -0,0 +1,1448 @@
+            - name: disease_type
+              label: Disease type
+              type: basic:string
+              description: |
+                General description of the disease.
+              required: false
+            - name: location
+              label: Location
+              type: basic:string
+              description: |
+                General description of the disease location.
+              required: false
+            - name: organ_part
+              label: Organ part
+              type: basic:string
+              description: |
+                General description of which part of the organ was affected.
+              required: false
+            - name: sampling
+              label: Sampling
+              type: basic:string
+              description: |
+                Select which method was used to obtain the sample.
+              required: false
+              choices:
+                - label: Biopsy
+                  value: biopsy
+                - label: Resection
+                  value: resection
+            - name: sampling_type
+              label: Sampling type
+              type: basic:string
+              description: |
+                Select where your sample was derived from, e.g. Primary (tumor) or Secondary (metastasis).
+              required: false
+              choices:
+                - label: Primary
+                  value: primary
+                - label: Secondary
+                  value: secondary
+            - name: biopsy_site
+              label: Biopsy site
+              type: basic:string
+              description: |
+                General description of where your sample was derived from,
+                e.g. stomach (primary site) or lymph nodes (secondary site), etc.
+              required: false
+            - name: grading
+              label: Grading
+              type: basic:string
+              description: |
+                General description of the disease stage.
+              required: false
+            - name: disease_status
+              label: Disease status
+              type: basic:string
+              description: |
+                Define status of the disease, e.g. progressive disease.
+              required: false
+            - name: disease_signature
+              label: Disease signature
+              type: basic:string
+              required: false
+            - name: metastases_sites
+              label: Metastases sites
+              type: basic:string
+              required: false
+            - name: number_of_metastases
+              label: Number of metastases
+              type: basic:integer
+              required: false
+            - name: mesenchymal
+              label: Mesenchymal
+              type: basic:string
+              description: |
+                Select if the cells are of mesenchymal type.
+              required: false
+              choices:
+                - label: "Yes"
```

Comment:
For all cases of "Yes" / "No", please check if the quotes are required. It suspect they are redundant.

Reply:
Tried running `test_api.py` with modified `general-clinical` DS (yes and no without quotes), but I get this error
```
RuntimeError:     VALIDATION ERROR: General clinical annotation
        path:       deque(['choices', 0, 'label'])
        message:    True is not of type 'string', 'integer', 'number'
        validator:  type
        val. value: ['string', 'integer', 'number']
```